content/2022/videookulografija.md

```diff
@@ -1,4 +1,561 @@
 ---
-title: Videookulografija
+title: Primena i implementacija algoritma za detekciju zenica baziranog na obradi slike
 summary: Projekat iz videookulografije rađen na letnjem kampu za stare polaznike 2022. godine od Anje Kovačev i Lenke Milojčević.
 ---
+**Autori** 
+
+
+Anja Kovačev, učenica IV razreda Tehničke škole u Kikindi
+
+
+Lenka Milojčević, učenica III razreda Gimnazije u Kraljevu
+
+
+**Mentori** 
+
+
+Milomir Stefanović, diplomirani inženjer elektrotehnke i računarstva, PFE
+
+
+Pavle Pađin, student Elektrotehničkog fakulteta u Beogradu
+
+
+
+## **1. Apstrakt**
+
+
+Tehnologija napreduje iz dana u dan, ali ipak osobe sa invaliditetom često nemaju mogućnost da komuniciraju, igraju video igre, da koriste kompjutere ili da se kreću. 
+Iz tih razloga odlučeno je da se ovaj rad bavi sistemom detekcije pokreta zenice koji bi dalje mogao da im pomogne u tim zadacima.
+Videookulografija je u današnje vreme najkorišćenija metoda za praćenje pokreta oka.
+U ovom radu, obradom slike ondnosno frejma videa u realnom vremenu, kao krajni rezultat se dobija detektovana zenica.
```

Comment:
Bojim se da niste skontale šta treba baš da se piše u apstraktu. Apstrakt treba da bude jak sažetak vašeg rada, što znači da treba da kažete koje ste metode koristili, ali većinski da pišete o rezultatima. Konkretno prvo kažete šta je cilj rada. Drugo koje su sve metode korišćene - u vašem slučaju je to jedna jedinstvena obrada slike pa je potrebno da kažete na čemu se zasniva, a kod vas je to Hafova transformacija. Nakon toga ide šta su rezultati i šta je krajnji zaključak. Evo simplifikovanog apstrakta.

Cilj ovog rada je dizajn hardvera i implementacija softvera za detekciju zenice sa ciljem njenog praćenja za primenu pokretanja kursora. Za dobijanje informacije o položaju zenice korišćena je web kamera. Sa slike sa kamere izvršena je segmentacija zenice, a zatim pomoću Hafove transformacije određen je centar i poluprečnik zenice. Korišćenjem polinomialne regresije reda N izvršena je transformacija koordinara zenice u koordinate kursora i posmatrane su greške prediktovane pozicije kursora u odnosu na željenu. U ovom radu dobijeno je da je radijalna greška tolika i tolika, što je dovoljno/nije dovoljno za realnu primenu. Kako bi se ovaj sistem koristio u realnom svetu, potrebno je uraditi to i to.